app-emulation/virt-manager/virt-manager-5.0.0.ebuild

```diff
@@ -80,6 +80,8 @@ pkg_preinst() {
 		rm -r "${ED}/usr/share/applications/virt-manager.desktop" || die
 		rm -r "${ED}/usr/bin/virt-manager" || die
 	fi
+
+	python_fix_shebang .
```

Comment:
We shouldn't do this in `pkg_preinst` as we want binary packages to be correct as they are. We should do it at the end of `src_install`.

But actually, the rest of what's happening here is a bit dubious too. We shouldn't be just deleting GUI bits from the image (already baked into the binpkg) just before merging...